src/modules/sign/containers/GithubLoginContainer.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+import { StateContext } from "../../../App";
+import { axiosInstance } from "../components/axiosApi";
+import GitSignIn from "../components/gitSignIn";
+
+const logIn = async function (token, dispatch, history, isGitlab) {
+    if(isGitlab){
+        try {
+            console.log("gitlab");
+            dispatch({ type: "LOGIN" })
+            const response = await axiosInstance.post(
+            "http://mentorpaired-staging-backend.herokuapp.com/gitlab_auth/",
+            { code: token }
+            );
+            console.log(response);
+            axiosInstance.defaults.headers["Authorization"] =
+            "JWT " + response.data.jwt.access;
+    
+            localStorage.setItem("access_token", response.data.jwt.access);
```

Comment:
Please move all the `localStorage.setItem` calls to a util function and then call the function from here. Would look cleaner that way and easier to debug.

Reply:
Okay, I'll make the changes.